packages/codefixes/src/codefixesMessages.json

```diff
@@ -0,0 +1,271 @@
+{
+  "addMissingAsync": {
+    "title": "Add Missing `async` Keyword",
```

Comment:
I would keep this array without converting it to ".generated" with `Diagnostic.***.code`, we don't need them in Rehearsal.